README.md

```diff
@@ -12,20 +12,18 @@ Currently the resource locks are applied to *resource groups* which have the fol
 The list could be extended by adding *|| contains(type, '<<*resource type*>>'))* to the [JSONPATH](https://github.com/hmcts/azure-resource-locks/blob/056dc8882431966269951abbef2f5dd9fd727e5e/scripts/enable-resource-locking.sh#L4) in the *enable-resource-locking.sh*
 
 # Pipeline jobs
-The enable resource locks pipeline jobs are scheduled to *run every 3 hours* for *CNP-DEV* , *AAT* and *Prod* environments. Each pipeline has been configured to use service connection(set as a pipeline variable) for the corresponding environments
-* [Enable-resource-locks-aat](https://dev.azure.com/hmcts/CNP/_build?definitionId=421)
-* [Enable-resource-locks-prod](https://dev.azure.com/hmcts/CNP/_build?definitionId=422)
+The enable resource locks pipeline jobs are scheduled to *run every 3 hours* for the *CNP-DEV*, *CNP-Prod*, *SDS-STG* and *SDS-PROD* environments. 
+Each pipeline has been configured to use the appropriate service connection automatically.
+* [Enable-resource-locks](https://dev.azure.com/hmcts/CNP/)
 
 
-The disable resource locks pipeline needs running manually by  specifying the corresponding service Connection and *resource-groups-list* (set as pipeline variables) for the *CNP-DEV* , *AAT* and *Prod* environments
-* On the [Disable-resource-locks](https://dev.azure.com/hmcts/CNP/_build?definitionId=423) build definition click on the *Run pipeline* button as shown below
+The disable resource locks pipeline must be run manually by selecting the name of a subscription from the dropdown and entering a list of resource groups to be targeted via the *resource-groups-list* pipeline variable.
+* On the [Disable-resource-locks](https://dev.azure.com/hmcts/CNP/_build?definitionId=423) build definition click on the *Run pipeline* button as shown in the image below:
```

Comment:
could we make resource groups a parameter please in disable?
 
Having to add a new variable is not a good UX, 

By adding that I think we can remove the need for all these pictures, they take up a lot of room and should be self explanatory, simple links and text should be enough what do you think?

see https://github.com/hmcts/azure-resource-locks/blob/090aae2775b8292fca59c3ad5045d74082fb0ae9/README.md

rendering is a bit off if you do think we need the pictures

Reply:
I think it's pretty self explanatory how to run it without them, I just updated them since there were some already there.

Re. resource groups. Will look into it.